nuitka/nodes/CtypesNodes.py

```diff
@@ -48,3 +49,15 @@ def replaceWithCompileTimeValue(self, trace_collection):
         trace_collection.onExceptionRaiseExit(BaseException)
 
         return self, None, None
+
+
+class ExpressionCtypesCIntCall(ExpressionCtypesCIntCallBase):
+    """Function reference ctypes.c_int"""
+
+    kind = "EXPRESSION_CTYPES_C_INT_CALL"
+
+    def replaceWithCompileTimeValue(self, trace_collection):
```

Comment:
For `ctypes.c_int(5)` values you would need one of these pseudo or actual compile time constant values and use compute them.

I am quoting an example here:

```
        return trace_collection.getCompileTimeComputationResult(
            node=self,
            computation=lambda: self.builtin_spec.simulateCall(given_values),
            description="Built-in call to '%s' computed."
            % (self.builtin_spec.getName()),
        )
```

For the call simulation to work, and produce a constant (or correct exception), quite a bit of work will be necessary. Compile time constants need quite a bit of handling, in code generation, in various constant query functions (mutable, is one, etc), data composer writing them, and in constant blob unstreaming.

There is plenty of examples to look at. I think `sys.version_info` is probably closest, although for searching others will be better. Another function to change is `makeConstantRefNode` needing a case.

Since these are going to be probably a lot of similar ones, maybe we need to make as much as possible of this more generic, but I do not know how really. Maybe go through it once, and then tell us how `c_long` could be made easier.